lib/rmt/mirror/base.rb

```diff
@@ -157,6 +157,7 @@ def cleanup_stale_metadata
 
   def move_files(glob:, destination:)
     FileUtils.mkpath(destination) unless Dir.exist?(destination)
+    FileUtils.rm_rf(Dir.glob(File.join(destination, '*')))
```

Comment:
Can we make this a conditional `clean_before`, so when running we need to set it explicitly. This would make the behavior clear, that the directory gets cleaned before moving.

Reply:
added the conditional parameter clean_before with default value to true.